tla/ccfraft.tla

```diff
@@ -1202,16 +1202,22 @@ QuorumLogInv ==
             \E j \in S :
                 IsPrefix(Committed(i), log[j])
 
+\* True if server i could receive a vote from server j based on the up-to-date check
+UpToDateCheck(i, j) ==
+    \/ MaxCommittableTerm(log[i]) > MaxCommittableTerm(log[j])
+    \/ /\ MaxCommittableTerm(log[i]) = MaxCommittableTerm(log[j])
+       /\ MaxCommittableIndex(log[i]) >= MaxCommittableIndex(log[j])
+
 \* The "up-to-date" check performed by servers
 \* before issuing a vote implies that i receives
 \* a vote from j only if i has all of j's committed
 \* entries
 MoreUpToDateCorrectInv ==
-    \A i, j \in Servers : i /= j =>
-        ((\/ MaxCommittableTerm(log[i]) > MaxCommittableTerm(log[j])
-         \/ /\ MaxCommittableTerm(log[i]) = MaxCommittableTerm(log[j])
-            /\ MaxCommittableIndex(log[i]) >= MaxCommittableIndex(log[j])) =>
-        IsPrefix(Committed(j), log[i]))
+    \A i \in { s \in Servers : state[s] = Pending } :
```

Comment:
`MoreUpToDateCorrectInv` will be trivially true after all servers transitions out of the `pending` state.

Reply:
My mistake, there’s a missing \not before the \in

Reply:
`\notin`?

Reply:
I switched to candidate instead of not pending, e37c664651c8049ad06b0fe2d1959f074b4d3388